store/cacheiterkvstore.go

```diff
@@ -0,0 +1,255 @@
+package store
+
+// TODO: Consider merge w/ tendermint/tmlibs/db/cachedb.go.
```

Comment:
make an issue & link it in the code

Reply:
I'll actually just do the whole thing.  1 moment...